Sources/DVR/Session.swift

```diff
@@ -211,9 +213,8 @@ open class Session: URLSession {
 
     private func persist(_ interactions: [Interaction]) {
         defer {
-            abort()
+            Session.didRecordCassetteCallback()
```

Comment:
the main change

Reply:
alternatively, it was possible to use a callback inside the DataTask, however this will break chained interactions (several requests per cassette) and make it less intuitive on how to catch recorded state.